skimage/transform/_warps.py

```diff
@@ -74,6 +74,13 @@ def resize(image, output_shape, order=1, mode=None, cval=0, clip=True,
     preserve_range : bool, optional
         Whether to keep the original range of values. Otherwise, the input
         image is converted according to the conventions of `img_as_float`.
+    anti_aliasing : bool
+        Whether to apply a Gaussian filter to smooth the image prior to
+        down-scaling. It is crucial to filter when resampling the image to
+        avoid aliasing artifacts.
+    anti_aliasing_stddev : {float, tuple of floats}
```

Comment:
My personal preference here would be `anti_aliasing_sigma`, because sigma is such a common parameter name for Gaussian filtering. But I'm not set on this, just a suggestion.

Reply:
Agreed, done.